### PR TITLE
Add epitools recipe

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- ~[ ] If static libraries are linked in, the license of the static library is packaged.~
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by drmatthews** [comment]
Hello @goanpeca - I'd rather not be added as a maintainer as I'm no longer actively involved with the epitools project.

**Comment by goanpeca** [comment]
Understood @drmatthews. Do you know who is currently active in the project?

**Comment by drmatthews** [comment]
@giuliapaci and @Pablo1990 have made commits and new release very recently.

**Comment by goanpeca** [comment]
@giuliapaci, @Pablo1990  do you agree with being added as maintainers?

**Comment by Pablo1990** [comment]
Fine by me! Thanks!

**Comment by giuliapaci** [comment]
Yes sounds good thanks!

**Comment by goanpeca** [comment]
This one is ready for review @conda-forge/staged-recipes :)

File: recipes/epitools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    - napari
+    - numpy
+    - pandas
+    - python >=3.9
+    - scikit-image >=0.20
+    - scipy
+
+test:
+  imports:
+    - epitools
```
Comment (jaimergp) [inline recipes/epitools/meta.yaml:35]:
```suggestion
    - epitools
    - epitools.main
```

File: recipes/epitools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    - epitools.main
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
```
Comment (jaimergp) [inline recipes/epitools/meta.yaml:41]:
```suggestion
    - pip
    - pyqt
```

**Pull request merged**